keras_contrib/applications/densenet.py

```diff
@@ -368,24 +388,25 @@ def __dense_block(x, nb_layers, nb_filter, growth_rate, bottleneck=False, dropou
     Returns: keras tensor with nb_layers of conv_block appended
     '''
 
-    concat_axis = 1 if K.image_dim_ordering() == "th" else -1
+    concat_axis = 1 if K.image_data_format() == 'channels_first' else -1
 
     x_list = [x]
 
     for i in range(nb_layers):
-        x = __conv_block(x, growth_rate, bottleneck, dropout_rate, weight_decay)
+        x = __conv_block(x, growth_rate, bottleneck,
+                         dropout_rate, weight_decay)
         x_list.append(x)
 
-        x1 = concatenate(x_list, axis=concat_axis)
-        #x = merge(x_list, mode='concat', concat_axis=concat_axis)
+        x = merge(x_list, mode='concat', concat_axis=concat_axis)
+        # x = concatenate(x_list, concat_axis)
```

Comment:
Remove commented line for now. Can be updated once a patch for the bug is merged onto Keras master.